Kinvey/Kinvey/Client.swift

```diff
@@ -256,6 +279,30 @@ open class Client: Credential {
         }
     }
     
+    /// Initialize a `Client` instance with all the needed parameters.
```

Comment:
This note should be more descriptive. Since this is now our recommended `init()` method, all the parameters should be explained in the reference doc.

Reply:
docs for parameters added